pages/content/amp-dev/documentation/courses/index.html

```diff
@@ -119,13 +118,20 @@ <h4>{{ _('Prerequisites') }}</h4>
 <section class="ap--video-slider">
   <h2 class="ap-o-video-slider-text">{{ _('Take the Video Courses') }}</h2>
   <p class="ap-o-video-slider-text">{{ _('All three courses can be now be taken by video on <a href="https://www.youtube.com/playlist?list=PLXTOW_XMsIDS45GB-eBV5s_M9EGIXMjI_" target="_blank">our YouTube channel!</a>') }}</p>
-  {% with %}
-  {% set youtube_video_ids = [
-    'qWpZbADJlgE',
-    'X3DybgZgN9k',
-    'v_HIPRR9E4Q'
-    ]
-  %}
-  {% include '/views/partials/video-carousel.j2' %}
-  {% endwith %}
+  [% set youTubeChannelPosts = []|importYouTubeChannel %]
```

Comment:
I think you need to make `importYouTubeChannel` configurable here as it should actually only show the latest coureses videos from [this playlist](https://www.youtube.com/playlist?list=PLXTOW_XMsIDS45GB-eBV5s_M9EGIXMjI_).

Reply:
If this can't be easily made dynamic, than revert back to previous solution. 😉